ibis/expr/api.py

```diff
@@ -845,6 +846,31 @@ def read_csv(sources: str | Path, **kwargs: Any) -> ir.Table:
     return con.read_csv(sources, **kwargs)
 
 
+def read_json(sources: str | Path, **kwargs: Any) -> ir.Table:
```

Comment:
do we want to mark this with `@experimental`?

Reply:
Yes. Yes I do.

Reply:
Done.